modules/ROOT/pages/_partials/commons/common-indexing.adoc

```diff
@@ -258,5 +277,15 @@ The query above will then produce the following output:
 
 The output demonstrates retrieval of both primary and secondary contact numbers listed as type `"mobile"`.
 
+Below is an example of using array indexes with dictionary values:
```

Comment:
What about `Here’s an example of creating an array index on a nested array containing dictionary values.`?

Reply:
Done!